### ISSUE TITLE
Unable to position the cursor using a screen reader and braille routing keys

### ISSUE THREAD

**Issue by mbrandl87** [microsoft/terminal#18733]
### Windows Terminal version

v1.23.10732.0

### Windows build number

10.0.26100.3476

### Other Software

vim inside wsl, bash, all over the place

### Steps to reproduce

Hi there,

I'm unable to move the cursor using braille routing keys e.g. on the line of the shell prompt or in vim on wsl. In regular text areas one can click the routing key above a character which then moves the cursor to the corresponding position. In wt it looks like every key just sends a left arrow, no matter how far on the right it is. in conhost nothing happens at all. 
When using showkey inside a wsl distro, it also reports an  [^D key code.

Using cygwin and its mintty it works, but showkey doesn't show any key then.

I tested using latest JAWS and NVDA.

Maybe there is just a setting I didn't come across so far?

Best
Martin 

### Expected Behavior

The cursor should move horizontally on the current line to the key I pressed

### Actual Behavior

It just moves one character to the left until it hits the line start and rings the bell

**Comment by DHowett** [comment]
Hey, thanks for reporting this! We didn't know such a feature existed... so we'll probably need some time to figure out how to handle it.
